pkg/resourceselector/blockdeviceselect/blockdeviceselector.go

```diff
@@ -0,0 +1,90 @@
+package blockdeviceselect
```

Comment:
can the package name be as follows:
`pkg/select/blockdevice/select.go`

Reply:
Changed